src/dmd/globals.d

```diff
@@ -158,6 +158,7 @@ struct Param
      */
 
     bool showGaggedErrors;  // print gagged errors anyway
+    bool contextPrintErrors;  // print errors with the error context (the error line in the source file)
```

Comment:
Perhaps `printErrorContext` is a better name?

Reply:
Renamed.